packages/schematics/angular/migrations/update-10/remove-solution-style-tsconfig_spec.ts

```diff
@@ -70,7 +70,6 @@ describe('Migration to remove "Solution Style" tsconfig', () => {
 
     // Create tsconfigs
     const compilerOptions = { target: 'es2015' };
-    createJsonFile(tree, 'tsconfig.json', { files: [] });
```

Comment:
This changes the behaviour of all test cases.

I think it's best to create a separate test case.

Ex:
```
  it(`should not error when 'tsconfig.json' doesn't exist`, async () => {
    tree.delete('tsconfig.json');
    const newTree = await schematicRunner.runSchematicAsync(schematicName, {}, tree).toPromise();
    expect(readJsonFile(newTree, 'tsconfig.json')['compilerOptions']).toBeTruthy();
    expect(newTree.exists('tsconfig.base.json')).toBeFalse();
  });
```